frontend/src/components/pages/AdminPage.tsx

```diff
@@ -20,21 +19,17 @@ import {
 import { useQuery } from "@apollo/client";
 
 import { AdminUser } from "../../types/UserTypes";
-import SideBar from "../common/Sidebar";
+import Navbar from "../common/Navbar";
 import Page from "../../types/PageTypes";
 import AdminUserTable from "../user-management/AdminUserTable";
 import AddAdminModal from "../user-management/AddAdminModal";
-import {
-  SettingsOutlineIcon,
-  AlertIcon,
-  SearchOutlineIcon,
-} from "../../assets/icons";
+import { AlertIcon, SearchOutlineIcon } from "../../assets/icons";
 import GET_USERS_BY_ROLE from "../../APIClients/queries/UserQueries";
 import SortTablePopover from "../common/SortTablePopover";
 
 const pages: Page[] = [
-  { title: "Assessments", url: "/", icon: SettingsOutlineIcon },
```

Comment:
Should there also be a `Dashboard` option that points to the admin page itself?

Reply:
I think `Database` refers to the dashboard? If that's what you mean?

Reply:
ohh i think i got confused cause i see these three options in Figma 
<img width="227" alt="image" src="https://user-images.githubusercontent.com/69697180/210287538-db4c8ef5-ccd6-4cb0-97e5-113985fcb88c.png">

Reply:
oh hmmm... i think there was some talk of adding an admin landing dashboard? but im not sure if we're still doing that 🤨, we can talk more  bout this during our next sync :P